### PR TITLE
New repo properties: lastPush & htmlURL

### PR DESCRIPTION
**Pull request by phatblat**
Added `Time` struct to hold the `NSDateFormatter` for parsing date strings.

### CODE REVIEW

File: OctoKit/Time.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
+//
```
Comment (pietbrauer) [inline OctoKit/Time.swift]: Can you remove the header comments please?

File: OctoKitTests/TimeTests.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//
```
Comment (pietbrauer) [inline OctoKitTests/TimeTests.swift]: Same here please.

**Comment by phatblat** [comment]
Done. Love that I can make simple changes like this with Git2Go

**Comment by pietbrauer** [comment]
Sweet :rocket:

**Comment by pietbrauer** [comment]
@phatblat Also exactly the reason why I started building it :grinning:

**Pull request merged**